src/applications/proxy-rewrite/teamsite-proxy.js

```diff
@@ -0,0 +1,96 @@
+const express = require('express');
+const fetch = require('node-fetch');
+const path = require('path');
+const cors = require('cors');
+
+const BUCKETS = require('../../site/constants/buckets');
+const ENVIRONMENTS = require('../../site/constants/environments');
+const HOSTNAMES = require('../../site/constants/hostnames');
+
+const VETS_WEBSITE_URL = process.env.VETS_WEBSITE_URL || 'localhost:3001';
+const TEAMSITE_PROXY_HOST = process.env.TEAMSITE_PROXY_HOST || 'localhost';
+const TEAMSITE_PROXY_PORT = process.env.TEAMSITE_PROXY_PORT || '3500';
+
+const PROD_BUCKET = BUCKETS[ENVIRONMENTS.VAGOVPROD];
+const PROD_DOMAIN = `https://${HOSTNAMES[ENVIRONMENTS.VAGOVPROD]}`;
+const PROD_BUCKET_REGEX = new RegExp(PROD_BUCKET.replace(/\./g, '\\.'), 'g');
+
+const CONTENT_TYPES = {
+  '.json': 'application/json',
+  '.css': 'text/css',
+  '.js': 'application/javascript',
+};
+
+const PROXY_REWRITE_ASSET_PATHS = ['/generated/', '/fonts/'];
```

Comment:
the teamsite app already has the ability to load assets from different environments. documentation [here](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/architecture/teamsite/). If this doesn't work anymore we should update the documentation and remove the deprecated loader code. 

> Since Teamsite content is not run through the vets-websitebuild process, testing changes on Teamsite requires running local or staging changes against the production pages. The proxy-rewriteapp supports targeting a specific environment for testing:
> 
> For example: https://staging.va.gov/health/?targetEnvironment=vagovstaging
> 
> The default behavior for https://staging.va.gov/health is to load the production assets. when the targetEnvironmentis set, the loader well delete the production nodes from the DOM and add script and style nodes for the target environment.
> 
> The injection script will support any of these environment names set as the targetEnvironment.
> 
> Caveats:
> 
> Style node deletions will prompt a rebuilding of the CSSOM but removing script nodes doesn't undo any JS that was run before the JS node was removed. For most cases this should not impact testing as the application specific code doesn't run but JS that is executed before the application (e.g. polyfills) will always run the production code first.